config/root/clusterrolebinding-tenancy-workspace-reader.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: system:kcp:authenticated:tenancy:reader
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:kcp:tenancy:reader
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: system:authenticated
```

Comment:
is this correct?

Reply:
I think a user can be `system:authenticated` but not in `SystemKcpClusterWorkspaceAccessGroup`

Reply:
that's intentional. That's one of the batteries in https://github.com/kcp-dev/kcp/pull/1688. In a real service this binding might not make sense. But for demo and development it does.